src/components/reviewCard.jsx

```diff
@@ -25,17 +38,24 @@ export function SingleReview() {
         <h2>{currentReview.title}</h2>
         <img src={currentReview.review_img_url}></img>
         <br />
+        <br />
         <p>{currentReview.review_body}</p>
         <br />
+        <br />
         <section className="Details">
           <p>Designer: {currentReview.designer}</p>
           <p>Category: {currentReview.category}</p>
           <p>Owner: {currentReview.owner}</p>
           <p>Created At: {currentReview.created_at}</p>
           <p>Votes: {currentReview.votes}</p>
-          <p>Comments: {currentReview.comments}</p>
+          <button className="comments-button" onClick={toggleComments}>
+            {showComments ? "Hide Comments" : "Show Comments"}
+          </button>
+          {showComments && <CommentSection comments={comments} />}
```

Comment:
Think about removing the bullet point beside the comment boxes, I'd suggest looking into adding a list-style to your CSS to make your page look cleaner 👍